packages/svelte2tsx/src/nodes/slot.ts

```diff
@@ -0,0 +1,158 @@
+import { Node, walk } from 'estree-walker';
+import MagicString from 'magic-string';
+import { attributeValueIsString, isMember } from '../utils/svelteAst';
+import TemplateScope from './TemplateScope';
+import { Identifier } from '../interfaces';
+
+function AttributeStrValueAsJsExpression(attr: Node): string {
+    if (attr.value.length == 0) return "''"; //wut?
+
+    //handle single value
+    if (attr.value.length == 1) {
+        const attrVal = attr.value[0];
+
+        if (attrVal.type == 'Text') {
+            return '"' + attrVal.raw + '"';
+        }
+    }
+
+    // we have multiple attribute values, so we know we are building a string out of them.
+    // so return a dummy string, it will typecheck the same :)
+    return '"__svelte_ts_string"';
+}
+
+export class SlotHandler {
+
+    constructor(private readonly str: MagicString, private readonly htmlx: string) { }
+
+    slots = new Map<string, Map<string, string>>();
+    resloved = new Map<Identifier, string>();
+    reslovedExpression = new Map<Node, string>();
+
+    reslove(identifierDef: Identifier, initExpression: Node, scope: TemplateScope) {
```

Comment:
`reslove` -> `resolve`. Happened in some other places as well 😄

Reply:
no wonder I keep not getting autocomplete 🤣